packages/order-experience-api/src/api/createController.ts

```diff
@@ -42,7 +43,17 @@ export class CreateController extends AbstractController<typeof requestSchema> {
     req: ValidatedRequest<typeof requestSchema>,
     res: Response
   ): Promise<any> {
-    const { items, customer } = req.body
+    const { items, customer, lastValidPurchaseDateTime } = req.body
+    let currentDateTime = new Date()
+
+    if (
```

Comment:
Could you create function to this controller which handles this validation logic.

So you can write automation tests easier in later